src/dev-to-git.ts

```diff
@@ -120,10 +121,22 @@ export class DevToGit {
 const devToGit = new DevToGit();
 devToGit
   .publishArticles()
-  .then(formatArticlePublishedStatuses)
-  .then(statuses => devToGit.logger(statuses))
-  .catch(err => {
+  .then(articles => ({ articles, text: formatArticlePublishedStatuses(articles) }))
+  .then(res => {
+    devToGit.logger(res.text);
+
+    res.articles.forEach(article => {
+      if (
+        article.updateStatus === UpdateStatus.ERROR ||
+        article.updateStatus === UpdateStatus.FAILED_TO_EXTRACT_FRONT_MATTER
```

Comment:
prob followup, but this is a very good case for a switch case, with the default being an error